### PR TITLE
Update team page.

### PR DESCRIPTION
**Pull request by nathanhammond**
Makes the teams page a bit more correct. Adds Tobias to the Ember CLI team. We need a 👍 from @Turbo87 before merging.

### CODE REVIEW

**Comment by Turbo87** [comment]
👍

File: data/team.yml [hunk 1]
```diff
@@ -2,128 +2,164 @@
...
+  github: https://github.com/nathanhammond
+  twitter: https://twitter.com/nathanhammond
+  image: nhammond.jpg
+  teams: 
+    - cli
+- name: Tobias Bieniek
+  github: https://github.com/Turbo87
+  image: tbieniek.jpg
+  teams: 
+    - cli
```
Comment (locks) [inline data/team.yml]: where's the trailing newline :trollface:

**Comment by locks** [comment]
After this merge, the page is visually identical confirm?

**Comment by nathanhammond** [comment]
Visually identical plus allowing of core team members to also be on subteams.

File: data/team.yml (hunk 1 above)
Comment (nathanhammond) [inline data/team.yml]: You mean you haven't set up an .editorconfig for this repo? :trollface:

Comment (acorncom) [inline data/team.yml]: PRs gladly accepted :trollface:

**Pull request merged**